lib/quantcast/index.js

```diff
@@ -156,38 +194,37 @@ Quantcast.prototype.completedOrder = function(track){
 };
 
 /**
- * Generate quantcast labels.
- *
- * Example:
- *
- *    options.advertise = false;
- *    labels('event', 'my event');
- *    // => "event.my event"
+ * Helper method to generate quantcast labels.
  *
- *    options.advertise = true;
- *    labels('event', 'my event');
- *    // => "_fp.event.my event"
- *
- * @param {String} type
- * @param {String} ...
+ * @param {String} prefix
+ * @param {Array} pageMeta
+ * @param {Array} ...
  * @return {String}
  * @api private
  */
 
-Quantcast.prototype._labels = function(type){
-  var args = Array.prototype.slice.call(arguments, 1);
-  var advertise = this.options.advertise;
+function createLabels(prefix, pageMeta){
+  var pageMeta = joinLabels(pageMeta, '.');
+  var labels = joinLabels([prefix, pageMeta], '.');
+  var customLabels = arguments[2];
 
-  if (advertise && 'page' == type) type = 'event';
-  if (advertise) type = '_fp.' + type;
+  if (customLabels) {
```

Comment:
(Seems like a facetious argument but someone might be constructing the `customLabels` array dynamically)